examples/apps/src/ble_bas_peripheral.rs

```diff
@@ -92,6 +102,7 @@ where
 ///
 /// spawner.must_spawn(ble_task(runner));
 /// ```
+// Uncertain, what the '.must_spawn()' does; is this a common Embassy pattern?
```

Comment:
Must spawn will panic if there is no free 'task handles' in the task pool. It's basically the same as spawn().unwrap().